packages/office-ui-fabric-react/src/components/Button/DefaultButton/DefaultButton.tsx

```diff
@@ -0,0 +1,27 @@
+/* tslint:disable:no-unused-variable no-unused-imports */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable no-unused-imports*/
+import {
+  BaseComponent,
+  getNativeProps,
+  anchorProperties,
+  buttonProperties
+} from '../../../Utilities';
+
+import { ButtonBase } from '../ButtonBase/ButtonBase';
+import { IButtonProps } from '../ButtonBase/ButtonBase.Props';
+import './DefaultButton.scss';
+
+export class DefaultButton extends BaseComponent<IButtonProps, any> {
```

Comment:
Once we deprecate Button we could rename it. But to keep Button working right now, it has to be given a new name.